src/middlewared/middlewared/plugins/system_advanced/config.py

```diff
@@ -195,20 +195,20 @@ async def do_update(self, data):
             )
 
             if original_data['boot_scrub'] != config_data['boot_scrub']:
-                await self.middleware.call('service.restart', 'cron')
+                await (await self.middleware.call('service.restart', 'cron')).wait(raise_error=True)
```

Comment:
Probably as a later improvement but might be nice to do this at the end now with `core.bulk` as user specified config will be completely applied and we won't stop before all the configuration changes have been applied. So what fails, can still fail but what works will at least work

Reply:
In other words, do you want to restart all services in parallel using `core.bulk`?

Reply:
I think that would be nicer but we can do that subsequently if that is better